source/features/profile-gists-link.tsx

```diff
@@ -56,16 +51,24 @@ async function appendTab(navigationBar: Element): Promise<void> {
 	navigationBar.append(link);
 	navigationBar.replaceWith(navigationBar);
 
-	select('.js-responsive-underlinenav .dropdown-menu ul')!.append(
-		createDropdownItem('Gists', user.url),
-	);
+	// There are two UnderlineNav items (responsive…) that point to the same dropdown
+	const overflowNav = select('.js-responsive-underlinenav .dropdown-menu ul')!;
+	if (!select.exists('[data-rgh-label="Gists"]', overflowNav)) {
```

Comment:
Avoids this issue:

<img width="391" alt="Screenshot 23" src="https://user-images.githubusercontent.com/1402241/215456258-e914c780-d898-4cc7-b38c-512fd543704a.png">